### DISCUSSION TITLE
BadTooManyPublishRequests when subscribe to nodes

### DISCUSSION THREAD

**Discussion by pavlo-yashchuk** [FreeOpcUa/python-opcua#1430] [Q&A] [answered]
Hi all! 
In the beginning, when subscribing to nodes I have error and warning logs from opcua
![image](https://user-images.githubusercontent.com/85494277/146942305-bba4630b-0bcd-4815-88c4-15a7f2aad4b1.png)
but after I get value if the value changes on nodes
what is that?

**Comment by AndreasHeine** [comment, marked as answer]
server? BadTooManyPublishRequests is an error response from the server!

**Reply by pavlo-yashchuk** [reply]
I have a client which creates multiple connections with 3 servers, don't have access to them
and after connecting create subscribe and then have these logs)

**Reply by pavlo-yashchuk** [reply]
Can I resolve it from the client side?

**Reply by AndreasHeine** [reply]
you also got an service fault...

**Reply by AndreasHeine** [reply]
what interval did you use?

could you share subscription settings ;)

**Reply by pavlo-yashchuk** [reply]
I see
I hope is that )) and I dont use intervals, node can be wring but try-except catch this error )
![image](https://user-images.githubusercontent.com/85494277/146958649-2b82c3e5-8216-4127-81b8-9fe5f6888538.png)
![image](https://user-images.githubusercontent.com/85494277/146958695-6092e17f-e29b-45af-b609-704698bfdb94.png)

**Reply by AndreasHeine** [reply]
and what server are they plc's? which vendor/product?

**Reply by AndreasHeine** [reply]
you are creating a subscription with 100ms publishing interval you could increase that if your server is a plc its more likly to be +=1000ms

**Reply by pavlo-yashchuk** [reply]
oh)) how do I know about this? Sorry about that )
I know only I need to get some id from specific nodes, and after some manipulation set a new value to another node )

**Reply by AndreasHeine** [reply]
set your logger on DEBUG and post the output!

**Reply by pavlo-yashchuk** [reply]
See only INFO, WARNING and ERROR logs 
![image](https://user-images.githubusercontent.com/85494277/146962743-048c5f56-5ea9-4f39-a09e-5ee875495df1.png)
![image](https://user-images.githubusercontent.com/85494277/146962588-63bf7c35-de91-4e52-b8ac-5e7f491203b4.png)
![image](https://user-images.githubusercontent.com/85494277/146962903-36158646-08b5-4abc-a7e1-8e6f488f71bf.png)
![image](https://user-images.githubusercontent.com/85494277/146962481-3318e302-29ce-4009-bdb0-c415752628ef.png)

**Reply by AndreasHeine** [reply]
logger.setLevel(logging.DEBUG)

**Reply by pavlo-yashchuk** [reply]
Sorry for the long answer, had a problem with getting logs)
[opcua_client.txt](https://github.com/FreeOpcUa/python-opcua/files/7761227/opcua_client.txt)
